pandas/tests/series/test_constructors.py

```diff
@@ -1810,6 +1810,18 @@ def test_constructor_bool_dtype_missing_values(self):
         expected = Series(True, index=[0], dtype="bool")
         tm.assert_series_equal(result, expected)
 
+    def test_constructor_int64_dtype(self):
```

Comment:
pls parameterize these. also add a case for uint64.

Reply:
Adding a case for uint64 will cause it to crash as this will implicitly type cast to int64 leading to an Assertion Error with uint64.

Parametrization is completed